BUILD.bazel

```diff
@@ -56,6 +56,11 @@ container_image(
         "/app/bazel-remote-base.binary",
         "--port=8080",
         "--dir=/data",
+
+        # This defaults to 127.0.0.1 which is inaccessible outside the container.
+        "--profile_host=",
+        # This is disabled by default.
+        "--profile_port=6060",
```

Comment:
I am not sure I understand the comment "This is disabled by default". It looks enabled to me?

Reply:
Updated the comments.

The (separate) http server with pprof endpoints is only started if --profile_port is specified.

Reply:
Thanks! Do you mind updating the README.md to also mention the necessity to add a -p mapping if one wants to access the profiling endpoint?

Reply:
Thanks- I forgot that. Pushed a fix. There are some other help text changes from the urfave/cli upgrade, added the non-profiling changes in a second commit.